internal/state/configuration.go

```diff
@@ -244,6 +246,7 @@ type hostPathRules struct {
 	rulesPerHost     map[string]map[string]PathRule
 	listenersForHost map[string]*listener
 	listeners        []*listener
```

Comment:
```
listeners        []*listener
``` 
would it make sense to rename it to httpsListeners or something like that?  The new code adds `listenersExist` variable, but just by looking at the names,  the new code poses a question: why can't we just check that len(listeners) > 0, why a separate boolean variable? the answer is `listeners` used only for HTTPS listeners, so I think it makes sense to rename that variable.

```
	listeners        []*listener
	listenersExist   bool
```

Reply:
yep, that makes sense